django-workload/django_workload/feed.py

```diff
@@ -10,6 +10,7 @@
 
 from .models import FeedEntryModel, UserModel
 from .users import suggested_users
+import json
```

Comment:
Note that this uses some C routines from `_json`.

Reply:
This was used before in the original workload.